README.md

```diff
@@ -3,6 +3,9 @@
 # Awesome Books
 
 > Awesome books is a simple project that displays new books when a user updates them.
+![screenshot](./images/spaadd.png)
+![screenshot](./images/spaList.png)
+![screenshot](./images/spacontact.png)
 
 
```

Comment:
- [ ] In this Readme file line 39, kindly change the project link for cloning from `personal-portfolio` to `awesomebooks`. Because this might confuse the users.

Reply:
Hello, @elyor-doniyorov thank you for this review I hadn't seen this.